intl-messageformat breaks due to missing dependency

**Which package?**
`intl-messageformat`

**Describe the bug**
Build breaks because of missing dependency.
This line https://github.com/formatjs/formatjs/blob/eaa0039c90533b09b0c03aa9dc9cd8c605405dba/packages/intl-messageformat-parser/src/skeleton.ts#L2 introduced a dependency on `@formatjs/intl-unified-numberformat` which is only declared in `devDependencies` and not in `dependencies` of the `package.json` of `intl-messageformat`.

**To Reproduce**
Steps to reproduce the behavior:

1. Add `intl-messageformat` version 7.7.1 as dependency.
2. Compile your project using the typescript compiler.
3. Get the error message `Cannot find module '@formatjs/intl-unified-numberformat'.`

**Expected behavior**
The compilation is successful.

--- a/packages/intl-messageformat-parser/package.json
+++ b/packages/intl-messageformat-parser/package.json
@@ -45,7 +45,7 @@
   "bugs": {
     "url": "https://github.com/formatjs/formatjs/issues"
   },
-  "devDependencies": {
+  "dependencies": {
     "@formatjs/intl-unified-numberformat": "^2.2.0"
   },
   "homepage": "https://github.com/formatjs/formatjs",

